### DISCUSSION TITLE
how can i solve the following error

### DISCUSSION THREAD

**Discussion by arisha00007** [coqui-ai/TTS#2679] [Archived]
> Setting up Audio Processor...
 | > sample_rate:22050
 | > resample:False
 | > num_mels:80
 | > log_func:np.log10
 | > min_level_db:0
 | > frame_shift_ms:None
 | > frame_length_ms:None
 | > ref_level_db:None
 | > fft_size:1024
 | > power:None
 | > preemphasis:0.0
 | > griffin_lim_iters:None
 | > signal_norm:None
 | > symmetric_norm:None
 | > mel_fmin:0
 | > mel_fmax:None
 | > pitch_fmin:None
 | > pitch_fmax:None
 | > spec_gain:20.0
 | > stft_pad_mode:reflect
 | > max_norm:1.0
 | > clip_norm:True
 | > do_trim_silence:False
 | > trim_db:60
 | > do_sound_norm:False
 | > do_amp_to_db_linear:True
 | > do_amp_to_db_mel:True
 | > do_rms_norm:False
 | > db_level:None
 | > stats_path:None
 | > base:10
 | > hop_length:256
 | > win_length:1024
 | > Found 377 files in /content/TTS/recipes/ljspeech/vits_tts/LJSpeech-1.1
 > Training Environment:
 | > Num. of CPUs: 2
 | > Num. of Torch Threads: 1
 | > Torch seed: 54321
 | > Torch CUDNN: True
 | > Torch CUDNN deterministic: False
 | > Torch CUDNN benchmark: False
 > Start Tensorboard: tensorboard --logdir=/content/TTS/recipes/ljspeech/vits_tts/vits_ljspeech-June-16-2023_08+00PM-d6581942

 > Model has 83059180 parameters

 > EPOCH: 0/1000
 --> /content/TTS/recipes/ljspeech/vits_tts/vits_ljspeech-June-16-2023_08+00PM-d6581942

> DataLoader initialization
| > Tokenizer:
	| > add_blank: True
	| > use_eos_bos: False
	| > use_phonemes: True
	| > phonemizer:
		| > phoneme language: en-us
		| > phoneme backend: gruut
| > Number of instances : 374
 ! Run is removed from /content/TTS/recipes/ljspeech/vits_tts/vits_ljspeech-June-16-2023_08+00PM-d6581942
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/trainer/trainer.py", line 1591, in fit
    self._fit()
  File "/usr/local/lib/python3.10/dist-packages/trainer/trainer.py", line 1544, in _fit
    self.train_epoch()
  File "/usr/local/lib/python3.10/dist-packages/trainer/trainer.py", line 1292, in train_epoch
    self.train_loader = self.get_train_dataloader(
  File "/usr/local/lib/python3.10/dist-packages/trainer/trainer.py", line 803, in get_train_dataloader
    return self._get_loader(
  File "/usr/local/lib/python3.10/dist-packages/trainer/trainer.py", line 767, in _get_loader
    loader = model.get_data_loader(
  File "/content/TTS/TTS/tts/models/vits.py", line 1618, in get_data_loader
    dataset.preprocess_samples()
  File "/content/TTS/TTS/tts/datasets/dataset.py", line 346, in preprocess_samples
    samples = self._compute_lengths(self.samples)
  File "/content/TTS/TTS/tts/datasets/dataset.py", line 298, in _compute_lengths
    audio_length = os.path.getsize(item["audio_file"]) / 16 * 8  # assuming 16bit audio
  File "/usr/lib/python3.10/genericpath.py", line 50, in getsize
    return os.stat(filename).st_size
FileNotFoundError: [Errno 2] No such file or directory: '/content/TTS/recipes/ljspeech/vits_tts/LJSpeech-1.1/wavs/\ufeff1.wav'

**Comment by jhwgh1968** [comment]
I'm going to guess there is a problem with your metadata file, which may be called `metadata.csv` or similar. It lists what wav files exist and their textual translations.

I'm guessing that `\ufeff1` is a Unicode character is not in the file path...
